templates/garb.cnf

```diff
@@ -0,0 +1,16 @@
+#{{ ansible_managed }}
+# Copyright (C) 2012 Codership Oy
```

Comment:
We can't keep documentation in config files up-to-date, therefore I think we shouldn't include it (unless automatically provided by the package). Users should refer to the actual documentation instead.

Without the documentation parts I don't think the file fulfills the threshold of originality so I think we can strip the copyright notice too.

The file should be as short as this:

```ini
GALERA_NODES="{{ mariadb.galera.cluster_nodes }}"
GALERA_GROUP={{ mariadb['my.cnf'].galera.wsrep_cluster_name }}
```

Btw. why are we having quotes around `GALERA_NODES` but none around `GALERA_GROUP`?

Reply:
Because the nodes are a more complex string conataining the list of nodes and at least MariaDB has got problems otherwise. I could add quotes to GALERA_GROUP as well, if you like.

Reply:
Fixed in 71620c4b9b0741f0b55625bca3f0eb9b75ee1812